src/app/features/activity-list/components/transaction-list/bitcoin-transaction/bitcoin-transaction.tsx

```diff
@@ -0,0 +1,11 @@
+import { BitcoinTx } from '@shared/models/transactions/bitcoin-transaction.model';
+
+import { BitcoinTransactionItem } from '@app/components/bitcoin-transaction-item/bitcoin-transaction-item';
+
+interface BitcoinTransactionProps {
+  transaction?: BitcoinTx;
+}
+
+export function BitcoinTransaction({ transaction }: BitcoinTransactionProps) {
+  return <BitcoinTransactionItem transaction={transaction} />;
+}
```

Comment:
Do we need this component, it does nothing

Reply:
I added it just to be consistent with stacks components for this feature